pages/apis/rest_api/pipelines.md.erb

```diff
@@ -563,6 +563,26 @@ Required scope: `write_pipelines`
 
 Success response: `204 No Content`
 
+## Add a webhook
+
+Create webhook for an existing pipeline.
```

Comment:
Maybe it's worth clarifying what kind of webhook? Is it configuring our system to send a webhook somewhere? Or creating a webhook on GitHub that will ping us on code push? (I think it's this one)